test/framework/kubernetes.go

```diff
@@ -143,7 +145,7 @@ func IsPodRunning(pod *corev1.Pod) bool {
 }
 
 // IsPodStatusConditionReady returns true if all pod's containers are ready (really running)
-func IsPodStatusConditionReady(pod *corev1.Pod) bool{
+func IsPodStatusConditionReady(pod *corev1.Pod) bool {
```

Comment:
VS automatically added this whitespace here, I think this is ok. Otherwise, let me know and I will revert this change.

Reply:
I am ok with that